src/pages/event-schema.ts

```diff
@@ -14,8 +14,20 @@ export const eventDateSchema = Joi.date()
     "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
   });
 
+export const endEventSchema = Joi.date()
```

Comment:
similar to the other comment: this schema describes the validation rules for the date only so I think it would be better to call it endDateSchema (like for the startDate one)